src/main/scala/com/github/vitalsoftware/scalaredox/models/Provider.scala

```diff
@@ -45,21 +45,19 @@ trait ProviderLike {
   def Location: Option[CareLocation]
 }
 
-trait Person {
-  def FirstName: String
-  def LastName: String
+trait WithContacts {
```

Comment:
I don't like this name: a Contact is the trait that corresponds to a person. Should be something like `WithContactDetails`